packages/dds/tree/src/test/utils.ts

```diff
@@ -1300,3 +1302,23 @@ export function moveWithin(
 ) {
 	editor.move(field, sourceIndex, count, field, destIndex);
 }
+
+/**
+ * Clones a batch of revertibles for a target branch.
```

Comment:
You should include a note about how the changes that the revertibles are targeting need to exist on the target branch.